server/src/main/java/org/elasticsearch/action/admin/cluster/node/info/NodesInfoRequest.java

```diff
@@ -281,12 +176,12 @@ public void writeTo(StreamOutput out) throws IOException {
      * from the nodes information endpoint. Eventually this list list will be
      * pluggable.
      */
-    enum Metrics {
+    public enum Metric {
         SETTINGS("settings"),
         OS("os"),
         PROCESS("process"),
         JVM("jvm"),
-        THREAD_POOL("threadPool"),
+        THREAD_POOL("thread_pool"),
```

Comment:
Ah, this change leads to a bwc problem. I'm going to make a separate PR to handle the bwc aspects of this, for simplicity.